src/components/Active-Button/page.module.css

```diff
@@ -0,0 +1,11 @@
+.ActiveButton {
+  height: 48px;
+  width: 324px;
```

Comment:
change to `padding: 20px 150px;` and delete height and width, padding is commonly used to define button sizes